evap/evaluation/migrations/0096_move_responsibilities_into_course.py

```diff
@@ -0,0 +1,35 @@
+from django.db import migrations
+
+
+def move_data_to_course(apps, _schema_editor):
+    Course = apps.get_model('evaluation', 'Course')
+    Contribution = apps.get_model('evaluation', 'Contribution')
+    for course in Course.objects.all():
+        responsibles = [contribution.contributor for contribution in Contribution.objects.filter(evaluation=course.evaluation, responsible=True)]
+        course.responsibles.set(responsibles)
+        course.save()
```

Comment:
In this migration, you do a `modelInstance.save()` after using `modelInstance.manyToManyField.set`, in some other migrations (especially in the commits before this), you do not. I did not find any documentation, but a bit of googling suggests that it is not required. I guess we want this unified?

(For all spots to look at, just Ctrl+F for ".set(" )